### PR TITLE
Update image service

### PR DESCRIPTION
**Pull request by pyby**
### Motivation and Context

The PR follows `SRGDataProvider` update regarding legacy image services removals: https://github.com/SRGSSR/srgdataprovider-apple/pull/44

### Description

- Update `SRGDataProvider dependency.
- Remove Artwork variant image url, which used a scaling aspect fitting in a black square.
- Artwork image uses `SRGImageSizeMedium` and `SRGImageVariantDefault`.
- Default image will now have 16:9 aspect ratio (`SRGImageVariantDefault` recommended size).

UI consequence:
- Same image is displayed in `SRGLetterboxView` and in the OS Control Center.

### Checklist

- [x] The branch should be rebased onto the `develop` branch for whole tests with nighties.
- [x] The documentation has been updated (if relevant).
- [x] The demo has been updated (if relevant).
- [x] Issues are linked to the PR, if any.

### CODE REVIEW

**Review by waliid — APPROVED** [review]

**Review by defagos — COMMENTED** [review]
Images do not appear in the control center for RTR, RSI, SRF, SWI (audio and video).

File: Sources/SRGLetterbox/SRGLetterboxService~ios.m [hunk 1]
```diff
@@ -579,7 +579,7 @@ - (void)updateMetadataWithController:(SRGLetterboxController *)controller
 - (NSURL *)artworkURLForController:(SRGLetterboxController *)controller withWidth:(SRGImageWidth)width
 {
     SRGMedia *media = controller.displayableMedia;
-    NSURL *artworkURL = SRGLetterboxArtworkImageURL(media.image, width, controller);
+    NSURL *artworkURL = SRGLetterboxImageURL(media.image, width, controller);
```
Comment (defagos) [inline Sources/SRGLetterbox/SRGLetterboxService~ios.m:582]: Issue with the implementation clearly visible here. The method was changed but the new one expects a size, not a width (both are integer-based enums so the compiler does not warn about the mismatch for some reason).

The result is that artwork image URLs are wrong. You should therefore either:

- Keep both width- and size-based APIs.
- Or use a size close to what is expected for the artwork (e.g. `SRGImageSizeMedium`) not a width.

**Review by defagos — CHANGES_REQUESTED** [review]
There is an issue with the implementation, artworks do not show up in the control center. See my comments for more information.

File: Sources/SRGLetterbox/SRGLetterboxService~ios.m (hunk 1 above)
Comment (pyby) [reply Sources/SRGLetterbox/SRGLetterboxService~ios.m]: Thanks a lot @defagos for the analyse and propositions. UI tests as no unit tests.

Artwork image has now switched to `SRGImageSizeMedium` and `SRGImageVariantDefault`.

**Review by defagos — APPROVED** [review]

**Pull request merged**